client/src/components/Search/SearchBox.scss

```diff
@@ -0,0 +1,192 @@
+$search-base-padding: 10px;
+$search-cancel-width: 52px;
+$search-filter-width: 35px;
+$search-enter-width: 52px;
+
+.search-box {
+  @include media-breakpoint-up(md) {
+    position: relative;
+    right: auto;
+
+    &.search-box__active {
+      right: 0;
+    }
+  }
+
+  &.search-box__active {
+    position: relative;
+    width: 100%;
+  }
+}
+
+.search-box__group {
+  opacity: 0;
+  overflow: hidden;
+  display: none;
+
+  .search-box__active & {
+    transition: opacity $transition-speed-fast;
+    overflow: visible;
+    opacity: 1;
+    display: block;
+    width: 100%;
+  }
+
+  .font-icon-search {
+    font-size: 25px;
+    position: absolute;
+    left: 20px;
+    top: 10px;
+  }
+}
+
+.search-box__enter {
+  display: none;
+  position: absolute;
+  right: $search-base-padding;
+  border: 1px solid $gray-100;
+  top: 16px;
+  padding: 0 4px;
+  font-size: $font-size-sm;
+  border-radius: 2px;
+  color: $gray-500;
+  user-select: none;
+
+  .search-box__has-filters & {
+    right: $search-base-padding + $search-filter-width;
+  }
+
+  .search-box__hideable & {
+    right: $search-base-padding + $search-cancel-width;
+  }
+
+  .search-box__has-filters.search-box__hideable & {
+    right: $search-base-padding + $search-filter-width + $search-cancel-width;
+  }
+
+  .search-box__has-focus & {
+    display: block;
+  }
+}
+
+// Search cancel
+.search-box__cancel {
+  right: 0;
+  top: 0;
+  position: absolute;
+  padding-right: 5px;
+  padding-left: 5px;
+  background-color: transparent;
+  height: 100%;
+  border-left: 1px solid $gray-200;
+  width: $search-cancel-width;
+  height: 54px;
+  border-radius: 0;
+  margin: 0;
+
+  &:hover {
+    background-color: $btn-secondary-bg;
+  }
+}
+
+// Expand filters panel
+.search-box__filter-trigger {
+  display: block;
+  border: 0;
+  margin: 0;
+  border-radius: 0;
+  position: absolute;
+  top: 10px;
+  outline: 0;
+  overflow: visible;
+  right: 5px;
+
+  .search-box__hideable & {
+    right: $search-cancel-width + 5px;
+  }
+
+
+  &:hover:not(:disabled):not(.disabled) {
+    background-color: transparent;
+
+    &:before {
+      color: $link-color;
+    }
+
+  }
+
+  &:active,
+  &:focus,
+  &:hover {
+    border: 0;
+    background-color: transparent;
+    box-shadow: none;
+  }
+
+  // Rotate ^ when filter-panel is open
+  &::before {
+    margin: 0;
+    transform: rotate(180deg) translateY(2px);
```

Comment:
You need to apply `display: inline-block;` for this transformation to work, also might be nice to use `.5turn` (`turn` is a little known valid unit) to make it a little bit clearer as to what the transformation is.

Reply:
That rotation style is a left over from the Asset Search component. The mock up doesn't actually have it. It might just be an omission. I'll need to have a chat with @clarkepaul to see what he wants.